docs/ROADMAP.md

```diff
@@ -24,23 +24,23 @@ Version numbering terminology:
 
 **GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
```

Comment:
A few lines earlier, add **Release Date:** Thursday, March 27th.... so we met our target!